source/2019-08-30-the-ember-times-issue-113.md

```diff
@@ -47,12 +48,9 @@ Check out the full tutorial by following along with the [demo repository](https:
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Thriving through the hype cycle: an Ember.js story 🌺](https://www.youtube.com/watch?v=ECkbVa0iC4k)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-
-<add your name to author list, top and bottom>
+Next to amazing Ember conferences, a lot of people are promoting Ember and sharing Ember wisdom at other conferences. [@locks](https://github.com/locks) told about `Thriving through the hype cycle: an Ember.js story` at [CommitPorto '19](https://commitporto.com/). This [talk](https://www.youtube.com/watch?v=ECkbVa0iC4k) gives great insights in how Ember as a framework is organised. Nothing too technical, but a nice peak in the kitchen of Ember. 
```

Comment:
Anne, I wasn't sure what you meant by "Next to amazing Ember conferences". Did you perhaps mean, "In addition to amazing Ember conferences", or "After amazing Ember conferences"?

```suggestion
Next to amazing Ember conferences, a lot of people are promoting Ember and sharing Ember wisdom at other conferences. [@locks](https://github.com/locks) told about `Thriving through the hype cycle: an Ember.js story` at [CommitPorto '19](https://commitporto.com/). This [talk](https://www.youtube.com/watch?v=ECkbVa0iC4k) gives great insights into how Ember, as a framework, is organised. Nothing too technical, but a nice peak in the kitchen of Ember. 
```